atlasdb-client/src/main/java/com/palantir/atlasdb/table/description/render/TableRenderer.java

```diff
@@ -199,7 +199,6 @@ protected void run() {
                 line("package ", packageName, ";");
                 line();
                 importRenderer.renderImports();
-                line();
```

Comment:
I think removing the line() in renderImports is better since I would expect this to only have one line (when you call line())